### ISSUE TITLE
Thesaurus load time increase in Windows 10

### ISSUE THREAD

**Issue by galaxnet** [gorzek/loomwords#12]
Since the last release, thesaurus load time has increased significantly, or appears to have based upon the onscreen text when running the CLI.

**Comment by gorzek** [comment]
Yes, it has. I plan to address a lot of performance/load time issues by producing a data cache on the first run which should make future runs faster to start. It might slightly slow down overall processing but that's the tradeoff.
